src/Microsoft.ML.StandardTrainers/Standard/SdcaBinary.cs

```diff
@@ -154,7 +154,7 @@ public abstract class SdcaTrainerBase<TOptions, TTransformer, TModel> : Stochast
         /// <summary>
         /// Options for the SDCA-based trainers.
         /// </summary>
-        public abstract class OptionsBase : TrainerInputBaseWithLabel
+        public abstract class OptionsBase : TrainerInputBaseWithWeight
```

Comment:
>TrainerInputBaseWithWeight [](start = 44, length = 26)

I would assume you check all mlContext.*Catalog extensions for SDCA trainers to have exampleWeightColumnName in it?

Reply:
yeap. i did verify all of the  `simple`  SDCA trainer extensions  have the `exampleWeightColumnName` parameter

---
In reply to: [269666016](https://github.com/dotnet/machinelearning/pull/3104#discussion_r269666016) [](ancestors = 269666016)